src/query/query_hash_scan.h

```diff
@@ -29,11 +29,43 @@
 
 #include "regu_var.hpp"
 
+#define MAKE_TUPLE_POSTION(tuple_pos, simple_pos, scan_id_p) \
+  do \
+    { \
+      tuple_pos.status = scan_id_p->status; \
+      tuple_pos.position = S_ON; \
+      tuple_pos.vpid = simple_pos->vpid; \
+      tuple_pos.offset = simple_pos->offset; \
+      tuple_pos.tpl = NULL; \
+      tuple_pos.tplno = 0; /* If tplno is needed, add it from scan_build_hash_list_scan() */ \
+    } \
+  while (0)
+
+/* kind of hash list scan method */
+enum hash_method
+{
+  HASH_METH_NOT_USE = 0,
+  HASH_METH_IN_MEM = 1,
+  HASH_METH_HYBRID = 2,
+  HASH_METH_HASH_FILE = 3 /* not used */
```

Comment:
not used? or not implement?

Reply:
'not implemented yet'. However, since no one knows the future, I just keep it 'not used'.